hiring/rationale.md

```diff
@@ -0,0 +1,15 @@
+# Rationale
+
+We believe that our hiring process should be fair and repeatable. What do we mean by this?
+
+## Fair
+
+Made is an equal opportunity employer that is committed to inclusion and diversity. There is a simple reason to this, we know that having people of all different walks of life provide us with new view points and better our ability to achieve our mission.
```

Comment:
I'd like this: "There is a simple reason to this: we know.." but I think @sowasred2012 has an aversion to colons